app/pages/lab/workflow.cjsx

```diff
@@ -318,7 +318,7 @@ EditWorkflowPage = React.createClass
               toggle = @handleSubjectSetToggle.bind this, subjectSet
               <tr key={subjectSet.id}>
                 <td><input type="checkbox" checked={assigned} onChange={toggle} /></td>
-                <td>{subjectSet.display_name}</td>
+                <td>{subjectSet.display_name} (#{@props.subjectSet.id})</td>
```

Comment:
don't use the props, just use `(#{subjectSet.id})`. 

You can test by running the front end locally https://github.com/zooniverse/Panoptes-Front-End/#development